active_learning_test/rb_streams.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+import time
+from collections import defaultdict
+from enum import Enum
+from itertools import chain, islice
+from typing import Iterable, Optional, Union
+
+import rubrix as rb
+from rubrix.client.models import Record
+from rubrix.client.sdk.commons.errors import NotFoundApiError
+
+
+class Priority(Enum):
+
```

Comment:
Very minor nitpick: I would drop the empty line here, and moreover, capitalize the constants. I think you are doing this already in [rubrix](https://github.com/recognai/rubrix/blob/f0652aa2532609a09897bb44d9888473e80d3279/src/rubrix/metrics/token_classification/metrics.py) as wll.

(No need to fix this in test project but maybe you will use this class for rubrix?)